app/controllers/group_health_controller.rb

```diff
@@ -62,7 +65,8 @@ def roles
   end
 
   def groups
-    respond(Group.from("((#{bund}) UNION (#{cantons}) UNION (#{abt_and_below})) as groups")
+    respond(Group.from("((#{bund}) UNION (#{cantons}) UNION (#{abt_and_below})) " \
+                       "AS #{Group.quoted_table_name}")
```

Comment:
Ich bin nicht sicher ob diese Änderung korrekt ist. Das hier ist ja ein Alias. Aber vielleicht wird der Alias sonstwo austauschbar mit dem Tabellennamen verwendet?

Reply:
wir könnten das alias (vielleicht) auch anders nennen. Wenn wir `groups` verwenden, muss der identifier für MySQL8 gequoted werden, weil dort `GROUPS` jetzt ein Keyword ist und sonst ein SQL-Syntaxfehler entsteht.

Reply:
Ach so, das ist der Hintergrund, alles klar. Ja dann würde ich den Alias einfach `relevant_groups` oder so nennen. Soweit ich sehe wird der Alias nirgends referenziert, aber ich glaube er ist nötig für `UNION` (war es zumindest als dieser Endpoint implementiert wurde).

Reply:
Ich lasse mir die Notfication mal in der INBOX und gehe das die Tage an. Es sollte ja schon getestet werden. Ich fürchte einfach, dass sich irgendwas in Rails auf das `groups` verlässt und will das nicht erst in production testen lassen :-)

Reply:
@carlobeltrame ich habe mal kurz versucht, das anzupassen und es kommen in der Tat Fehler, die auf das Brechen von Konvention hindeuten. Ich würde es dann lieber so lassen.